modules/openapi-generator/src/main/resources/php-symfony/composer.mustache

```diff
@@ -19,22 +19,24 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": ">=8.0.2",
```

Comment:
Would it be breaking to keep also >= 7.4 supported?
`>=7.4`
`>=7.4.0|>=8.0.2`

Reply:
The thing is: symfony6 only support `php>=8.0.2`.  
But now that I think about it, I can currently not see an obvious reason why symfony5 should not work.

I'll try it in the next days.

Reply:
There were no conflicts. So it works with symfony 5.

I'm not sure why I decided it needs to target symfony6 only in the first place.